source/integrations/zapier.rst

```diff
@@ -139,8 +139,13 @@ Cannot connect a Mattermost account
   a. Possible Solution: The **Client Secret** might be incorrect. Verify this value in **Main Menu** > **Integrations** > **OAuth 2.0 Applications**, or check with your System Admin.
 
   .. image:: ../images/zapier-error4.png
+  
+4. **"Error Invalid client id"**
+  a. Possible Solution: The **Client ID** and/or **Client Secret** might have trailing spaces in them when copy and pasted into the form. Check for trailing spaces in the **Client ID** and **Client Secret** fields and resubmit.
 
-4. **"Mattermost needs your help: We couldn't find the requested app"**
+  .. image:: ../images/zapier-trailing-space-error.png
```

Comment:
@cvitter on your branch can you please also add the image file to the source/images folder with the same title as you used in the path here?